### ISSUE TITLE
The colored text over white background on the landing page is hard to read (and doesn't meet accessibility guidelines)

### ISSUE THREAD

**Issue by hydrocheck** [hydroshare/hydroshare#4444]
The contrast ratio is very low for the colored text over the white background.  This causes readibility issues (and accessibility issues), particularly for small and low weight text (e.g., the "indable" in the FIND acronym image).  It appears as though this contrast issue spans multiple components (e.g., also on https://help.hydroshare.org/ and the Home page)
<img width="999" alt="004" src="https://user-images.githubusercontent.com/43424907/147899809-308a5e01-fb17-4bc8-ace8-9965130241f6.png">
<img width="1201" alt="004-01" src="https://user-images.githubusercontent.com/43424907/147899812-b140752a-0544-4f6d-afd9-a06f545c557e.png">
<img width="1440" alt="004-02" src="https://user-images.githubusercontent.com/43424907/147899815-7c5edefb-1855-4b33-bdd6-9e1c9d5cbec0.png">
